warehouse/redshift/redshift_test.go

```diff
@@ -53,6 +53,17 @@ func TestRedshiftIntegration(t *testing.T) {
 		}), fmt.Sprintf("Failed dropping schema %s for Redshift", handle.Schema))
 	})
 
+	require.NoError(t, testhelper.SetConfig([]warehouseutils.KeyValue{
```

Comment:
Currently, we don't have a clear way of running tests with configurations since during tests rudder server runs in an isolated environment.
In `SetConfig` we are making and API call which sets up the config and reload the hot reloadables.

Reply:
If it fails, no need to proceed since setting the configuration is required.

Reply:
The problem with the existing test is that we get coverage, but do not actually test the logic.

If you replace the query with: `AND 1=1`, it will still pass. It is useful for doing some syntax & sanity checking. Ensuring that the code will not err/fail/panic, but it is not complete.

Reply:
Good point. Let's discuss more around this once later.